### PR TITLE
MINOR: Provide valid examples in README page.

### PR DESCRIPTION
**Pull request by kamalcph**
- `testMetadataUpdateWaitTime` method is removed from MetadataTest class.
- travis-ci.org will be shut down in some time. All the interactions are now via travis-ci.com.
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kamalcph** [comment]
@chia7712 ping for review

**Comment by kamalcph** [comment]
@chia7712 @omkreddy 
ping for review

File: tests/README.md [hunk 1]
```diff
@@ -97,26 +97,26 @@ Examining CI run
 ----------------
 * Set BUILD_ID is travis ci's build id. E.g. build id is 169519874 for the following build
 ```bash
-https://travis-ci.org/apache/kafka/builds/169519874
```
Comment (chia7712) [inline tests/README.md:100]: The travis CI was already removed from kafka. maybe we can remove those docs. @ijuma WDYT?

Comment (ijuma) [reply tests/README.md:100]: Sounds good.

Comment (kamalcph) [reply tests/README.md:100]: Removed the doc.

**Comment by kamalcph** [comment]
@chia7712 
Can you please review this patch?

**Review by chia7712 — APPROVED** [review]
LGTM

**Comment by kamalcph** [comment]
@ijuma Can you please review and merge this patch? Thanks!

**Comment by kamalcph** [comment]
@ijuma @showuon 
Please take a look

**Review by showuon — APPROVED** [review]
LGTM!

File: tests/README.md (hunk 1 above)
Comment (showuon) [reply tests/README.md:100]: Agree to remove the unused travis CI doc.

**Comment by showuon** [comment]
@kamalcph , thanks for the patch!

**Pull request merged**